tools/c7n_left/c7n_left/core.py

```diff
@@ -233,16 +233,16 @@ def run(self) -> bool:
             for p in self.policies:
                 if not self.match_type(rtype, p):
                     continue
-                result_set = self.run_policy(p, graph, resources, event)
+                result_set = self.run_policy(p, graph, resources, event, rtype)
                 if result_set:
                     self.reporter.on_results(result_set)
                     found = True
         self.reporter.on_execution_ended()
         return found
 
-    def run_policy(self, policy, graph, resources, event):
+    def run_policy(self, policy, graph, resources, event, resource_type):
         event = dict(event)
-        event.update({"graph": graph, "resources": resources})
+        event.update({"graph": graph, "resources": resources, 'resource_type': resource_type})
```

Comment:
Mix of single quote and double quote is a bit jarring

Reply:
noted, hmm, maybe its time to enable black string normalization